src/server.js

```diff
@@ -62,7 +63,7 @@ export default function server({app, path, urls, origin, i18ndir, reducers, rout
       referer: origin
     });
     const cookie = new CookieDough(req);
-    const history = createHistory(req.originalUrl);
```

Comment:
When I passed `{basename: req.originalUrl}` option to `createHistory()`, `Link`'s path on server side was broken. (like `/ja/ja?tags[]=a`)